pkg/controller/operands/operand_test.go

```diff
@@ -81,4 +87,69 @@ var _ = Describe("Test operator.go", func() {
 			Expect(obj.Spec.Config.FilesystemOverhead.Global).Should(BeEquivalentTo("55"))
 		})
 	})
+
+	Context("Test addCrToTheRelatedObjectList", func() {
+		It("Should return error when apiVersion, kind and name missing", func() {
+			hco := commonTestUtils.NewHco()
+			req := commonTestUtils.NewReq(hco)
+			found := &cdiv1beta1.CDI{}
+
+			operand := genericOperand{Scheme: scheme.Scheme}
+			err := operand.addCrToTheRelatedObjectList(req, found)
+			Expect(err).To(HaveOccurred())
+			Expect(err.Error()).To(ContainSubstring("object reference must have, at a minimum: apiVersion, kind, and name"))
+		})
+
+		It("Should add into the list when it is missing", func() {
+			hco := commonTestUtils.NewHco()
+			req := commonTestUtils.NewReq(hco)
+			found := &cdiv1beta1.CDI{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "CDI",
+					APIVersion: "cdi.kubevirt.io/v1beta1",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cdi-kubevirt-hyperconverged",
+				},
+			}
+
+			operand := genericOperand{Scheme: scheme.Scheme}
+			err := operand.addCrToTheRelatedObjectList(req, found)
+			Expect(err).ToNot(HaveOccurred())
+
+			foundRef, err := reference.GetReference(operand.Scheme, found)
+			Expect(err).To(BeNil())
```

Comment:
For errors, please use `Expect(somethingThatMayCauseError).ToNot(HaveOccurred())` - for a better readability of the test.